e2e/docker/Dockerfile.prometheus-static-data

```diff
@@ -1,10 +1,11 @@
-FROM docker.io/prom/prometheus:v3.1.0
+FROM docker.io/prom/prometheus:v3.2.1
 
 USER nobody
 
 # data from ~ 2025-02-18 14:30 to 2025-02-18 17:15
-COPY provisioning/prometheus/data.zip /prometheus/data.zip
-RUN unzip /prometheus/data.zip 
-RUN rm /prometheus/data.zip 
-
-COPY provisioning/prometheus/prometheus.yml /etc/prometheus/prometheus.yml
```

Comment:
Removed that last copy statement in favor of using docker volumes for the prometheus config file.

Still kept it for the unzipping and copying of the `data` directory